v2/cmd/zlint/main.go

```diff
@@ -210,3 +220,139 @@ func setLints() (lint.Registry, error) {
 
 	return lint.GlobalRegistry().Filter(filterOpts)
 }
+
+func outputSummary(zlintResult *zlint.ResultSet, longSummary bool) {
+	var sortedLevels []int
+	resultCount := make(map[lint.LintStatus]int)
+	resultDetails := make(map[lint.LintStatus][]string)
+	lintLevelsAboveThreshold := make(map[int]lint.LintStatus)
+	// Set the threashold under which (inclusive) events are not
+	// counted
+	threshold := lint.Pass
+
+	// Make the list of lint levels that matter
+	for _, i := range lint.StatusLabelToLintStatus {
+		if i <= threshold {
+			continue
+		}
+		lintLevelsAboveThreshold[int(i)] = i
+	}
+	// Set all of the levels to 0 events to they are all displayed
+	// in the -summary table
+	for _, level := range lintLevelsAboveThreshold {
+		resultCount[level] = 0
+	}
+	// Count up the number of each event
+	for key, value := range zlintResult.Results {
```

Comment:
small nit: I think this loop could be better if you used more descriptive var names. Something like:

```go
name, result := range zlintResults.Results {
   ...
}
```

Reply:
fixed in b3c3f69